include/gatekeeper_net.h

```diff
@@ -166,6 +166,23 @@ struct gatekeeper_if {
 	/* The maximum packet lifetime. */
 	uint8_t         ipv6_default_hop_limits;
 
+	/*
+	 * According to the 82599 datasheet, a receive descriptor is
+	 * a data structure that contains the receive data buffer address
+	 * and fields for hardware to store packet information. Upon receipt
+	 * of a packet for this device, hardware stores the packet data into
+	 * the indicated buffer and writes the length, status and errors to
+	 * the receive descriptor.
+	 *
+	 * Each packet buffer is indicated by a descriptor.
+	 */
+	uint16_t        gatekeeper_num_rx_desc;
```

Comment:
1. Rename `gatekeeper_num_rx_desc` to `num_rx_desc`. It's not a global parameter since the field belongs to `struct gatekeeper_if` instead of `struct net_config`.

2. Do the same for field `gatekeeper_num_tx_desc` below.